### DISCUSSION TITLE
راه اندازی مرزبان روی سرور های ایرانی

### DISCUSSION THREAD

**Discussion by M03ED** [Gozargah/Marzban#145] [Show and tell]
سلام.
ممکنه شما دوستان مثل من نیاز داشته باشید مرزبان رو روی یک سرور داخلی راه اندازی کنید ولی متوجه شدید به خاطر تحریم ها نمیتونید این کار رو انجام بدید.
برای انجام این کار نیاز به تغییر dns داریم و باید از سرویس هایی مثل شکن داریم.
با زدن دستور resolvectl status در لینوکس وضعیت کارت های شبکه ( چه مجازی و چه فیزیکی ) رو میتونید ببینید.
![image](https://user-images.githubusercontent.com/50927468/228294402-01a70dc5-f5a6-40f3-8a06-413bacfe60f5.png)
کارت شبکه اصلی رو پیدا میکنیم ( برای بنده اخرین گزینه هست )
با دستور Ctrl+C از این صفحه خارج میشیم و با دستور <resolvectl dns <interface> <dns address1> <dns address2 اقدام به تغییر DNS می کنیم.
![image](https://user-images.githubusercontent.com/50927468/228297020-dc5fe46f-a93d-4a25-a507-ae381fd7a9a7.png)
تمام ، حالا میتونید خیلی راحت مرزبان رو از طریق داکر نصب کنید .
توجه استفاده از vpn روی سرور های داخلی صرفا برای استفاده شخصی مناسب هست ، اکیدا از فروش چنین سرویسی خودداری کنید چون احتمال انجام فیشینگ و فعالیت های غیرقانونی دیگه ای هست که گریبان گیر شخصی که سرور به نامش ثبت شده میشه.

پیروز و سربلند باشید.
به امید آزادی.

**Comment by M03ED** [comment]
یکی از دلایلش میتونه دسترسی به سایت های ایران اکسس باشه 
دلایل دیگه هم میتونه داشته باشه

**Comment by KAJOOSH** [comment]
اگر همچنان با خطای 403 رو به رو شدید از دستور ذیل استفاده کنید

`sudo systemctl restart systemd-resolved`

**Comment by mahsamcse** [comment]
درود
ببخشید الان من تعدادی درخواست دارم از افراد خارج از کشور که می‌خوان به سایت های ایرانی (مثل بانکها) دسترسی داشته باشند
ینی ریسک زیادی داره من این سیستم رو راه بندازم بدم بهشون ؟
راهی نیست بشه نظارت یا پیشگیری کرد از سواستفاده احتمالی از سرور ایران که بنام خودم هست ؟
چون اینطور که شما گفتین نمیشه اعتماد کرد برامون داستان میشه
مگه اینکه سرور ایران به نام خودشون که در خارج هستن بگیریم که البته اگه بازم مشکلی پیش بیاد و پرونده قضایی تشکیل بشه به نام اون فرد دیگه نمیتونه بیاد ایران .

**Reply by MinitorMHS** [reply]
برای شمایی که میخوای سرور بگیری بله ریسک داره
البته من یه سرویس دهنده خارجی دیدم که سرور ایران ارائه میداد و آیپی های فیلتر رو هم باز می کرد و مشکلی نداشت
بنظرم اگه از اونجا vps تهیه کنی مشکلی نداره ولی خب خیلی از فیشر ها با اسم همین کارای بانکی و اینجور چیزا دنبال سرور ایران هستن
دیگه به خودت بستگی داره
https://exservers.net/vps-iran

**Comment by MinitorMHS** [comment]
اینم من اضافه کنم که این کارو میتونید با دستور
`nano /etc/network/interfaces`

انجام بدید و توی خط آخر DNS ها رو ادیت کنید و بعد با Ctrl+x سیو کنید
بعد کافیه دستور زیر رو اجرا کنید و تمام
`service networking restart`

![IMG_20240131_164016_045.jpg](https://github.com/Gozargah/Marzban/assets/99525672/3991b04a-766c-4fad-b2b7-de571b2becfa)

**Comment by sarboy1371** [comment]
با این روش دیگه داکر نصب نمی شه .لطفا روش جدید رو بذارید

**Comment by irantechh** [comment]
این روش کار نیمکنه
curl: (28) SSL connection timeout
